src/BloomBrowserUI/bookEdit/js/readerToolsModel.ts

```diff
@@ -1063,6 +1063,21 @@ class ReaderToolsModel {
       returnVal.push(new DataWord(words[i]));
     }
 
+    // inform the user if the list was truncated
+    var accordion: Window = iframeChannel.getAccordionWindow();
+    var msgDiv: JQuery = $(accordion).find('#allowed-word-list-truncated');
+
+    // We are reducing the max by 10 here in case there were a couple of duplicates that were removed after the list
+    // was truncated, causing the total number of words to now be slightly less than the max.
+    if (words.length < model.maxAllowedWords - 10) {
```

Comment:
Another idea: another recent PR changed wording, at least, to indicate that what is shown somewhere is 'new decodable words". Not sure whether that also applies to stages defined by word list, but it would seem consistent for it to do so. Then we'd only need to retrieve one stage's worth for this box, and there would be no possibility of duplication between stages, and overflow would be less likely...and quite possibly it would be useful to help the author focus on the new words being learned.